verifier/pom.xml

```diff
@@ -281,6 +281,37 @@
   </build>
 
   <profiles>
+
+    <profile>
+      <id>build-docker-image</id>
+      <activation>
+        <property>
+          <name>env.DOCKER_HOST</name>
+        </property>
+      </activation>
+      <build>
+        <plugins>
+          <plugin>
+            <groupId>io.fabric8</groupId>
+            <artifactId>fabric8-maven-plugin</artifactId>
+            <version>${fabric8-maven-plugin.version}</version>
+            <executions>
+              <execution>
+                <id>build-docker-image</id>
+                <goals>
+                  <goal>build</goal>
+                </goals>
+                <phase>package</phase>
+                <configuration>
+                  <mode>kubernetes</mode>
```

Comment:
Not sure whether we should enforce kubernetes mode, but I'm fine with that for now until I have verified that image streams are working nicely, too.

Reply:
Yeah.. would be nice to detect openshift env, and enable openshift style image builds.